guides/developer/getting-set-up.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: single
+title: "Getting Set Up for Spinnaker Development"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+This page describes the steps a Developer should take to fetch Spinnaker's codebase
+and get set up to work on it.
+
+# System Requirements
+
+This guide assumes you have access to a machine with a minimum specification of:
+
+- At least 16 GB of RAM, ideally more
+- A 4 core CPU
+
+Spinnaker is developed on Linux. This guide uses Halyard which requires one of the following
+flavors of Linux:
+
+- Ubuntu 14.04, 16.04 or 18.04
+
+# Fetching Spinnaker's codebase
+
+The following steps will install Spinnaker's management tool, Halyard, fetch Spinnaker's
+codebase, and perform just enough configuration to get Spinnaker up and running.
+
+Each of these steps will take between 5 and 30 minutes to finish and may require you to
```

Comment:
What is the 30 minute step?

Reply:
Configuring the LocalGit took me about 25 minutes on a run-through yesterday and I figure it'll take someone who's new a bit longer.  There are quite a few `apt-get`s to sit through if you don't have any of the cli tools installed plus the manual process of visiting and forking the 10 services.  Admittedly it wasn't the most focused run-through.